homeassistant/components/solarlog/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "title": "Solar-Log",
+    "step": {
+      "user": {
+        "title": "Define your Solar-Log connection",
+        "data": {
+          "host": "The hostname or ip-address of your Solar-Log device (without http://)",
```

Comment:
Can we see if `http://` is added inside the config flow, and add it if it is not ?

Reply:
Good idea. I’ve added this.